include/pistache/cookie.h

```diff
@@ -73,18 +96,22 @@ class CookieJar {
 
     private:
         Storage::const_iterator it_;
+        HashMapCookies::const_iterator it_2;
+        Storage::const_iterator end_; // we need to know where main hashmap ends. 
     };
 
     CookieJar();
 
     void add(const Cookie& cookie);
+    void removeCookie(const std::string& name);  // ADDED LATER
```

Comment:
The comment should be "Unimplemented", not "[to be] added later".  In fact, I would comment out the entire line until you have an implementation.